wca-regulations.md

```diff
@@ -19,7 +19,7 @@ Original source of the WCA regulations: [www.worldcubeassociation.org/regulation
 WCA Regulations in [PDF format](link:pdf)
 
 ### Source
-Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/cubing/wca-documents).
+Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/thewca/wca-documents).
```

Comment:
Ah, good catch! That url I created doesn't even work. I'll change this to <https://github.com/thewca/wca-regulations>.

Wow you guys still haven't fixed #361? FOR SHAME. I'll do it.